stdlib/public/Platform/CMakeLists.txt

```diff
@@ -17,11 +17,7 @@ add_swift_target_library(swiftDarwin ${SWIFT_STDLIB_LIBRARY_BUILD_TYPES} IS_SDK_
 
     SWIFT_COMPILE_FLAGS -Xfrontend -disable-objc-attr-requires-foundation-module "${SWIFT_RUNTIME_SWIFT_COMPILE_FLAGS}"
     LINK_FLAGS "${SWIFT_RUNTIME_SWIFT_LINK_FLAGS}"
-    TARGET_SDKS ALL_APPLE_PLATFORMS
-
-    # This is overly conservative, but we have so few API notes files that
-    # haven't migrated to the Swift repo that it's probably fine in practice.
-    DEPENDS copy_apinotes)
```

Comment:
I think this is still relevant; this is about the build you're in the process of doing rather than the install.

Reply:
I've updated this to keep the dependency when building the standard library with the compiler, but also allow building standalone without the dependency.